test/test_elliptic.jl

```diff
@@ -88,14 +88,14 @@ end
         q = map(p -> iszero(p) ? ones(typeof(p)) : (Ω + I) * normalize(p), P)
         p = @inferred LCSParameters(1.0, 3*max(step(xspan), step(yspan)), combine, 60, 0.5, 1.5, 1e-4)
 
-        vortices, singularities = constrainedLCS(q, p; outermost=true, verbose=false)
+        vortices, singularities = constrainedLCS(q, p; outermost=true, verbose=false,debug=false)
```

Comment:
Here and below: do we need the explicit `debug`? I thought `false` this is the default.